solr/core/src/java/org/apache/solr/cli/DeleteTool.java

```diff
@@ -92,7 +94,13 @@ public void runImpl(CommandLine cli) throws Exception {
     SolrCLI.raiseLogLevelUnlessVerbose(cli);
     String solrUrl = SolrCLI.normalizeSolrUrl(cli);
 
-    try (var solrClient = SolrCLI.getSolrClient(solrUrl)) {
+    System.out.println("HEre are teh options:");
```

Comment:
weird casing on this message, and typo ;)

Reply:
So, for some reason, for me, I am not getting the `-credentials` parameters in this tool.   I wish I had a junit test instead of just bats...   This is debugging...

Reply:
so, my unit test passes, but not the bats test.  sigh.